### ISSUE TITLE
discoverRegex need improvement

### ISSUE THREAD

**Issue by bfwg** [cgiffard/node-simplecrawler#197]
```js 
var crawler = new Crawler('http://www.cgiffard.com');
```
I try to test the crawler on your website :P and I got this.
`start fetching http://www.cgiffard.com/background.jpg`
`start fetching http://www.cgiffard.com/%22background.jpg%22;`
After I look into this a little bit I found out that `"background.jpg";` is passing our Regex.

I did a bit debugging and find out that 
```js
/\s?(?:href|src)\s?=\s?[^"'][^\s>]+/ig
```
Is the reason cause `"background.jpg;"` to pass.

I change it to 
```js
/\s?(?:href|src)\s?=\s?[^"'\s][^\s>]+/ig
```

Now it works like a charm.
Hope it helps.

**Comment by fredrikekelund** [comment]
@bfwg could you submit this as a PR? Thanks!

**Comment by fredrikekelund** [comment]
Merged fix in #203
